runtime/src/snapshot_utils.rs

```diff
@@ -54,8 +54,11 @@ pub const TMP_SNAPSHOT_ARCHIVE_PREFIX: &str = "tmp-snapshot-archive-";
 pub const MAX_BANK_SNAPSHOTS_TO_RETAIN: usize = 8; // Save some bank snapshots but not too many
 pub const DEFAULT_MAX_FULL_SNAPSHOT_ARCHIVES_TO_RETAIN: usize = 2;
 pub const DEFAULT_MAX_INCREMENTAL_SNAPSHOT_ARCHIVES_TO_RETAIN: usize = 4;
-pub const FULL_SNAPSHOT_ARCHIVE_FILENAME_REGEX: &str = r"^snapshot-(?P<slot>[[:digit:]]+)-(?P<hash>[[:alnum:]]+)\.(?P<ext>tar|tar\.bz2|tar\.zst|tar\.gz)$";
-pub const INCREMENTAL_SNAPSHOT_ARCHIVE_FILENAME_REGEX: &str = r"^incremental-snapshot-(?P<base>[[:digit:]]+)-(?P<slot>[[:digit:]]+)-(?P<hash>[[:alnum:]]+)\.(?P<ext>tar|tar\.bz2|tar\.zst|tar\.gz)$";
+pub const FULL_SNAPSHOT_ARCHIVE_FILENAME_PREFIX: &str = r"snapshot-";
+pub const INCREMENTAL_SNAPSHOT_ARCHIVE_FILENAME_PREFIX: &str = r"incremental-snapshot-";
+pub const FULL_SNAPSHOT_ARCHIVE_FILENAME_REGEX: &str = concat!("^", FULL_SNAPSHOT_ARCHIVE_FILENAME_PREFIX, "(?P<slot>[[:digit:]]+)-(?P<hash>[[:alnum:]]+)\.(?P<ext>tar|tar\.bz2|tar\.zst|tar\.gz)$");
+pub const INCREMENTAL_SNAPSHOT_ARCHIVE_FILENAME_REGEX: &str = concat!("^", INCREMENTAL_SNAPSHOT_ARCHIVE_FILENAME_PREFIX,"(?P<base>[[:digit:]]+)-(?P<slot>[[:digit:]]+)-(?P<hash>[[:alnum:]]+)\.(?P<ext>tar|tar\.bz2|tar\.zst|tar\.gz)$";
+pub const LOCAL_SNAPSHOT_FILE_NAME: &str = ".local_snapshot";
```

Comment:
nit: Since we already use `FILENAME` in constants, can this one be `LOCAL_SNAPSHOT_FILENAME` instead?

Reply:
Fixed